### ISSUE TITLE
Carriers: Add up stored units in the unit storage tab

### ISSUE THREAD

**Issue by Basilisk3** [FAForever/fa#5344]
Right now, each unit that is stored inside another unit is counted individually in the unit storage tab (right under the constriction tab). For example, if there are 50 ASFs in a carrier, there are 50 individual ASF icons shown in the carrier’s storage tab. This is confusing and makes it hard to discern, how many units are stored. 

Inspired by the recently added feature of mobile factories.

**Comment by Garanas** [comment]
We could batch them up, like we do with a regular selection

**Comment by Garanas** [comment]
I agree, not something we can add in for the next patch however. The file in question is, uhh - a beast to work with 😄

**Comment by HUSSAR-mtrela** [comment]
@Garanas I will add up stored units in carries but I think we should not add up stored units in transporters because you can give individual orders to unload engineers/tanks

**Comment by Garanas** [comment]
I agree, this would just apply to carriers

**Comment by clyfordv** [comment]
https://github.com/FAForever/fa/pull/6112

**Comment by 4z0t** [comment]
:eyes:
